src/main/java/io/aiven/kafka/connect/common/output/jsonwriter/ValueBuilder.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.output.jsonwriter;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.json.JsonConverter;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+class ValueBuilder implements OutputFieldBuilder {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    private final JsonConverter converter;
+
+    public ValueBuilder() {
+        converter = new JsonConverter();
+        // TODO: make a more generic configuration
+        converter.configure(Map.of("schemas.enable", false, "converter.type", "value"));
+    }
+
+    /**
+     * Takes the {@link SinkRecord}'s value as a JSON.
+     *
+     * @param record        the record to get the value from
+     * @throws DataException when the value is not actually a JSON
+     * @return JsonNode     Value transformed to any JSON value
+     */
+    @Override
+    public JsonNode build(final SinkRecord record) throws IOException {
+        Objects.requireNonNull(record, "record cannot be null");
+        Objects.requireNonNull(record.valueSchema(), "value schema cannot be null");
```

Comment:
it seems like right now, for JsonValue `record.valueSchema() != null` is not required. It is possible, that messages in Kafka doesn't include schema but serialized to a valid json

Reply:
This is true, but will `JsonConverter.fromConnectData` be able to handle this?

Reply:
Well, at least in integration test I could see that it reads json string, transform it successfully to JsonNode -> Bytes, and Than use it to write values. At least it did not throw an Exception, when a schema is disabled.

It seems like it could lead to a problem, if schema is enabled for JsonConverter (in OutputWriter, but disabled for Kafka events)

Reply:
E.g. if I pass `null` instead of `level1Schema` [here](https://github.com/aiven/aiven-kafka-connect-commons/blob/977767388c2197546671c90e87b60c800c5c780f/src/test/java/io/aiven/kafka/connect/common/output/WithMetadataJsonLinesOutputWriterTest.java#L70), it'll fail with this:

```
org.apache.kafka.connect.errors.DataException: Mismatching schema.

	at org.apache.kafka.connect.json.JsonConverter.convertToJson(JsonConverter.java:677)
	at org.apache.kafka.connect.json.JsonConverter.convertToJsonWithoutEnvelope(JsonConverter.java:571)
	at org.apache.kafka.connect.json.JsonConverter.fromConnectData(JsonConverter.java:320)
	at io.aiven.kafka.connect.common.output.jsonwriter.ValueBuilder.build(ValueBuilder.java:57)
	at io.aiven.kafka.connect.common.output.jsonwriter.JsonLinesOutputWriter.writeEntry(JsonLinesOutputWriter.java:85)
	at io.aiven.kafka.connect.common.output.jsonwriter.JsonLinesOutputWriter.writeFields(JsonLinesOutputWriter.java:77)
	at io.aiven.kafka.connect.common.output.jsonwriter.JsonLinesOutputWriter.writeLastRecord(JsonLinesOutputWriter.java:68)
```

Reply:
OTOH, the schema seems to be always provided at this stage anyway.

Reply:
Let's keep it as is.